### ISSUE TITLE
Carousel target dots should be larger

### ISSUE THREAD

**Issue by StephDriver** [openlibhums/hourglass#353]
## Report Issue
**Describe the inaccessible feature**
Minimum target size is 24 x 24 see #339 
The dots for moving the carousel are smaller than this minimum (15px).

**Which users are impacted?**
 (delete as applicable)
 1. Everyone

**To Reproduce**
Steps to reproduce:
1. example - [our team page](https://www.openlibhums.org/site/our-team/)

**Screenshots**
<img width="396" alt="image" src="https://github.com/BirkbeckCTP/hourglass/assets/5330770/4d856d87-1a58-4828-b933-0ee654202519">

## Preferred Solution
Increase the size of these dots to 24 x 24 px.

## Front-end Issues (only)
If the issue is front-end specific please add the following details:

**Desktop (please complete the following information):**
 - MacOS Chrome
